### DISCUSSION TITLE
I would like to see support for localization.

### DISCUSSION THREAD

**Discussion by L4Ph** [civitai/civitai#392] [Ideas]
As the number of non-English speaking users (especially Japanese users) is increasing, I am thinking that I would like to have a standard feature of Next.js or even something like i18next.
Since it is open source, it is also possible to use Crowdin for free, so I would like your opinion on this!

**Comment by ProfaneServitor** [comment]
Me agree

**Comment by comradekingu** [comment]
@Katsuyuki-Karasawa
Crowdin was never free for open source projects. It is only no cost for "non-commercial open source projects".
Moreover the terms and agreements of signing over shared rights to works means it isn't gratis either.
There is no upside to Crowdin as a tool either.
Happy to help if you use Weblate.
